### ISSUE TITLE
Pre-Training on cutom dataset

### ISSUE THREAD

**Issue by adoshi085** [D-X-Y/ResNeXt-DenseNet#21]
Hi @D-X-Y,
Nice work!
I have a query that I want to pre-train on a large fashion dataset using Resnext arch.

I would want to ask a few questions in this regards-
1. To start with I think I first need to define a new function in resnext.py file with the depth, cardinality and the base_width of my choice. Right?
2. I would want to use 101, 32, 4d. Would this be possible if I stick to the same CifarResNext class or do I need to create a separate one?
3. The dataset is in the same manner as Imagenet Dataset is (Train and Val folders with subfolders inside them for each class). So not many tweaks required except for a few in imagenet_train.py file. Right?
4. I would then be using this model to fine-tune my own fashion dataset by using fastai. Any pointers here from you will help

Thanks in advance

Regards,

**Comment by D-X-Y** [comment]
1, yes,
2, you can use the same one.
3, Only need to change the number of classes in `https://github.com/D-X-Y/ResNeXt-DenseNet/blob/master/imagenet_train.py#L54` with the #classes of your new dataset, and input a different root path : https://github.com/D-X-Y/ResNeXt-DenseNet/blob/master/imagenet_train.py#L23
4, For fine-tuning, you might need to tune the learning rate.

Best Regards,
